src/plugins/intel_gpu/src/plugin/transformations/clamp_fp16_output.cpp

```diff
@@ -28,7 +32,12 @@ ClampFP16Output::ClampFP16Output() {
     auto in1 = any_input(as_value_predicate(class_other_than<v0::Constant>()));
     auto matmul_m = wrap_type<v0::MatMul>({in0, in1}, all_of({type_matches(ov::element::f16), consumers_count(1)}));
     auto reshape_m = wrap_type<v1::Reshape>({matmul_m, any_input()}, all_of({type_matches(ov::element::f16), consumers_count(1)}));
-    auto softmax_input_m = std::make_shared<Or>(ov::OutputVector{reshape_m, matmul_m});
+    auto add_m = wrap_type<v1::Add>({matmul_m, any_input()}, all_of({type_matches(ov::element::f16), consumers_count(1)}));
+    auto multiply_m = wrap_type<v1::Multiply>({matmul_m, any_input()}, all_of({type_matches(ov::element::f16), consumers_count(1)}));
+    auto subtract_m = wrap_type<v1::Subtract>({matmul_m, any_input()}, all_of({type_matches(ov::element::f16), consumers_count(1)}));
+    auto divide_m = wrap_type<v1::Divide>({matmul_m, any_input()}, all_of({type_matches(ov::element::f16), consumers_count(1)}));
+    auto eltwise_m = std::make_shared<Or>(ov::OutputVector{add_m, multiply_m, subtract_m, divide_m});
```

Comment:
wrap_type allows multiple types to be passes:
```cpp
auto eltwise_m = wrap_type<v1::Divide, v1::Add, v1::Multiply, v1::Subtract>({matmul_m, any_input()}, all_of({type_matches(ov::element::f16), consumers_count(1)}));
```

but in this case it can be simplified to:
```cpp
auto eltwise_m = wrap_type<op::util::BinaryElementwiseArithmetic>({matmul_m, any_input()}, all_of({type_matches(ov::element::f16), consumers_count(1)}));
```

Reply:
Among `BinaryElementwiseArithmetic`, only the four ops (`Add`, `Subtract`, `Multiply`, `Divide`) support gemm+eltwise fusion in gpu plugin side, so I updated it as the first suggestion.